docs/module_guides/storing/vector_stores.md

```diff
@@ -99,6 +100,7 @@ maxdepth: 1
 /examples/vector_stores/SimpleIndexDemo.ipynb
 /examples/vector_stores/SupabaseVectorIndexDemo.ipynb
 /examples/vector_stores/TairIndexDemo.ipynb
+../../../llama-index-integrations/vector_stores/llama-index-vector-stores-tidbvector/examples/tidb_vector.ipynb
```

Comment:
hmm, I don't think this will work, the docs needs to be in the `docs` folder.

Should we move the notebook to `docs/examples/vector_stores`?

Reply:
Ok, I will fix this

Reply:
Fixed